### ISSUE TITLE
Thanks for your great work！Do you have a corresponding article for the code?

### ISSUE THREAD

**Issue by flyxuexi** [xgyutu/NeXt-Stereo#1]

**Comment by xgyutu** [comment]
The paper for this code is about to be uploaded as a preprint. Thank you for your attention.

**Comment by flyxuexi** [comment]
> The paper for this code is about to be uploaded as a preprint. Thank you for your attention.

Do you have a pdf version of the article? Also, I would like to ask what the model's epe indicator is in sceneflow. Thank you very much！

**Comment by xgyutu** [comment]
You can wait for the preprint of the paper manuscript. Our model achieves an EPE metric of 0.49 on SceneFlow.

**Comment by flyxuexi** [comment]
> You can wait for the preprint of the paper manuscript. Our model achieves an EPE metric of 0.49 on SceneFlow.

Thanks a lot

**Comment by flyxuexi** [comment]
> You can wait for the preprint of the paper manuscript. Our model achieves an EPE metric of 0.49 on SceneFlow.

 May I ask you what the training strategy for your sceneflow dataset is? I think your readme is fastacv's strategy. I trained for 26 rounds and finally got epe=0.77.Thanks a lot！

**Comment by xgyutu** [comment]
Hello. You can try training the various parts of the model in stages, such as training the refinement part of the model in the second stage and fine-tuning the rest of the model. Or check whether you have used the pre-trained weights for the backbone.
